LoggerProj/Logger/BaseLoggerMixins.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+
+namespace Logger
+{
+    public static class BaseLoggerMixins
+    {
+        public static void Error(this BaseLogger baseLogger, string message, params object[] args)
+        {
+            LogLevel logLevel = LogLevel.Error;
+
+            if (baseLogger is null)
+            {
+                throw new ArgumentNullException(nameof(baseLogger));
+            }
+            else
+            {
+                baseLogger.Log(logLevel, String.Format(message, args));
```

Comment:
When possible, favor using C# keywords over the class name. 
```suggestion
                baseLogger.Log(logLevel, string.Format(message, args));
```